BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1210,6 +1209,11 @@ IEnumerator FireReloadableMissile()
                 wpm.heatTarget = TargetSignatureData.noTarget;
             }
             ml.TargetPosition = transform.position + (multiLauncher ? vessel.ReferenceTransform.up * 5000 : transform.forward * 5000); //set initial target position so if no target update, missileBase will count a miss if it nears this point or is flying post-thrust
+            if (ml == null)
+            {
+                //Debug.LogWarning($"[BDArmory.MissileLauncher]: Missile {part.name} destroyed during initialization, MissileLauncher now null!");
+                yield break;
+            }
             ml.MissileLaunch();
```

Comment:
This would prevent the NREs from the missile getting destroyed during the coroutine, but I'm not sure this is the best way to go about it. It'd prevent having to encapsulate the entire section in a if (ml != null), but the other option would be, what, a second coroutine to terminate this one if ml suddenly = null?

Reply:
Yeah, I think this approach is probably the best, however, I've applied it further up after the `yield return wait ...` in dev.
The problem is that `ml` is not `null`, but the `gameObject` is.